subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CheckstyleReports.java

```diff
@@ -34,7 +34,7 @@
      * @return The checkstyle HTML report
      * @since 2.10
      */
-    SingleFileReport getHtml();
+    CheckstyleHtmlReport getHtml();
```

Comment:
The target was to be consistent with the `FindbugsReports` API and its internal implementation `FindbugsReportsImpl` (which declare the type `FindBugsXmlReport` on the `getXml()` method):
- https://github.com/gradle/gradle/blob/REL_2.11/subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugsReports.java#L35
- https://github.com/gradle/gradle/blob/REL_2.11/subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/internal/FindBugsReportsImpl.java#L40

The type was switched back to `SingleFileReport` on the PR rework.